cmd/registry/patch/artifact.go

```diff
@@ -206,6 +212,24 @@ func applyArtifactPatch(ctx context.Context, client connection.Client, content *
 	return err
 }
 
+// populateIdAndKind inserts the "id" and "kind" in the supplied json bytes
```

Comment:
Nit: I think there's a missing word "fields" between "kind" and "in". Also, comments that are sentences should end with periods.